# Project Application to *Funding Institution*

### Title of the project:
Unveiling the mechanisms of Disease X

### Applicant & Principal Investigator:
Prof. Ree Searcher

### Contact:
ree.searcher@biodata.pt

### Host Institution:
BioData.pt

### Start date:
January 1st, 2020

### Duration:
36 months

## Abstract:

The cause of Disease X has been recently discovered to be a virus, phage X, which infects normal gut bacteria and leads them to become virulent and cause chronic intestinal infection. Although non-fatal, this disease has been spreading rapidly in Europe, with costs in health-care reaching the tens of millions of Euros.

This project aims to uncover the mechanisms of disease X by sequencing phage X and studying the effects of its infection in human gut microbiota at the population and molecular level. We will assess which bacterial taxa are infected by phage X and what effect the infection has on the relative abundance of the various taxa, as well as what effect the infection has on the infected taxa at the gene expression level.

The project will be a key step towards improving treatment for Disease X, and potentially being able to cure it.

## Research plan and method summary:

The project will be divided into four activities:
1. Sample collection
2. Phage X sequencing
3. 16S sequencing
4. Metatranscriptomics

In the sample collection activity, we will define a study group of volunteer disease X patients, numbering no less than 20, and a control group comprising their close relatives, 1-2 per patient. We will collect stool samples from each of the volunteers.

In the Phage X sequencing activity, we will carry out DNA sequencing of the stool samples and assemble the genome of Phage X. In order to facilitate the assembly while enabling the reliable identification of sequence variants, we will combine the higher quality but short read sequencing technology of the Illumina NextSeq 500 sequencer with the long read but lower quality technology of the Nanopore MinION sequencer.

In the 16S sequencing activity, we will do microbial diversity analysis of the stool samples based on DNA sequencing of the 16S rRNA gene, to assess the impact of Phage X infection of the diversity and relative abundance of gut bacteria taxa. This sequencing will be carried out in Illumina NextSeq 500 sequencers.

In the metatranscriptomics activity, we will assess the effect of Phage X infection at the gene expression level through RNA sequencing of the samples. This sequencing will be carried out in Illumina NextSeq 500 sequencers.

## Predicted data outputs:

1. ***Sample collection***

- Data:

  - Sample identification table (< 1 MB)

- Metadata:

  - Patient metadata (< 1 MB)

  - Sample collection metadata (< 1 MB)

2. ***Phage X sequencing***

- Data:

  - Raw FASTQ sequencing data - NextSeq (60 MB)

  - Raw FASTQ sequencing data - MinION (1 GB)

  - Assembled Phage X genome (< 1 MB)

- Metadata:

  - Sample preparation & sequencing metadata - NextSeq (< 1 MB)

  - Sample preparation & sequencing metadata - MinION (< 1 MB)

  - Assembly metadata (< 1 MB)

3. ***16S sequencing***

- Data:

  - Raw FASTQ sequencing data (15 GB)
 
  - Biome tables (< 1 MB)
 
- Metadata:
 
  - Sample preparation & sequencing metadata - NextSeq (< 1 MB)

4. ***Metatranscriptomics***

- Data:

  - Raw FASTQ sequencing data (120 GB)

  - RNAseq count tables (< 1 MB)
  
  - Differential expression test results (< 1 MB)

- Metadata:

  - Sample preparation & sequencing metadata - NextSeq (< 1 MB)
